addon/components/file-card.hbs

```diff
@@ -0,0 +1,38 @@
+<div
+  class="file-card"
+  style="position: relative;"
+  {{! template-lint-disable no-inline-styles}}
+  ...attributes
+>
+  {{#if (and this.isRemovable @onDelete)}}
+    <button
+      type="button"
+      class="au-c-close"
+      {{on "click" (fn @onDelete @file)}}
+    >
+      <AuIcon @icon="cross" @size="large" />
+      <span class="au-u-hidden-visually">Verwijderen</span>
+    </button>
+  {{/if}}
+
+  <div class="au-o-box au-o-box--small au-c-card">
+    <AuContent @skin="small">
+      <AuHeading @level="4" @skin="6">
+        {{@file.filename}}
```

Comment:
Maybe the file shouldn't be passed in directly so we aren't stuck with a specific file model? (or being forced to build the file object before passing it in).